deploy/cr.yaml

```
@@ -27,7 +27,7 @@ spec:
 #  pause: false
   updateStrategy: SmartUpdate
   upgradeOptions:
-    versionServiceEndpoint: https://check.percona.com
+    versionServiceEndpoint: https://check.percona.com/versions
```

lets have correct value in the CR
```suggestion
    versionServiceEndpoint: https://check.percona.com
```